packages/blade/src/tokens/theme/createTheme.ts

```diff
@@ -0,0 +1,376 @@
+              focus: foregroundOnBrandColorPaymentLight,
+              active: foregroundOnBrandColorPaymentLight,
+            },
+            secondary: {
+              default: foregroundOnSurfacePaymentLight,
+              hover: foregroundOnSurfacePaymentLight,
+              focus: foregroundOnSurfacePaymentLight,
+              active: foregroundOnSurfacePaymentLight,
+            },
+          },
+          icon: {
+            primary: {
+              default: foregroundOnBrandColorPaymentLight,
+              hover: foregroundOnBrandColorPaymentLight,
+              focus: foregroundOnBrandColorPaymentLight,
+              active: foregroundOnBrandColorPaymentLight,
+            },
+            secondary: {
+              default: foregroundOnSurfacePaymentLight,
+              hover: foregroundOnSurfacePaymentLight,
+              focus: foregroundOnSurfacePaymentLight,
+              active: foregroundOnSurfacePaymentLight,
+            },
+          },
+        },
+      },
+    },
+  };
+
+  return overrideTheme({
+    baseThemeTokens: paymentTheme,
+    overrides: lightThemePaymentsOverrides,
+  });
+};
+
+/**
+ *
+ * @param brandColors - The brand colors to use to override the banking theme
+ * @description Overrides the banking theme with the brand colors passed in
+ * @returns The dark theme tokens with the custom brand colors
+ */
+const getDarkTheme = (brandColors: Color['chromatic']['azure']): ThemeTokens => {
+  // Select the most readable color to use as the foreground color on top of brand color
+  // For example: On Primary Button where the background color is brand color, the text color should be either dark or light depending on which is more readable on top of that brand color
+  const foregroundOnBrandColorBankingDark = tinycolor
+    .mostReadable(
+      brandColors[800],
+      [globalColors.neutral.navyGrayDark[800], globalColors.neutral.navyGrayDark[0]],
+      WCAG2ContrastOptions,
+    )
+    .toHexString();
+
+  // Select the most readable color to use as the foreground color on top of surface color
+  // For example: On Secondary Button where the background color is surface color, the text color should be either the brand color or light color depending on which is more readable on top of that surface color
+  const foregroundOnSurfaceBankingDark = tinycolor.isReadable(
+    globalColors.neutral.navyGrayDark[1100],
+    brandColors[400],
+    WCAG2ContrastOptions,
+  )
+    ? brandColors[400]
+    : globalColors.neutral.navyGrayDark[0];
+
+  // Override the banking theme onDark with the brand colors
+  const darkThemeBankingOverrides = {
+    colors: {
+      onDark: {
+        brand: {
+          primary: {
+            300: brandColors.a100,
+            400: brandColors.a200,
+            500: brandColors[400],
+            600: brandColors[500],
+            700: brandColors[600],
+            800: brandColors[900],
+          },
+          gray: {
```

Comment:
for using overrideTheme function, don't we need all the keys in the object even though the ones that are not being overridden? will the keys missing will be replaced by its default value? or will it just remove those keys from final output?

Reply:
Replaced by its default value. Tried this too by replacing only hover value of action background